server/controller/orders/updateOrderState.js

```diff
@@ -59,9 +64,10 @@ const updateOrderState = async (req, res, next) => {
         if (resourcesPrice < 0 || Number.isNaN(resourcesPrice)) {
           throw boomify(400, 'please enter resources price');
         }
-
-        providerPriceHour = (await getPriceHourProvider(providerId)).rows[0]
-          .price_hour;
+        const { rows: [providerData]} = await getProviderDataById(providerId);
+        // providerPriceHour = (await getPriceHourProvider(providerId)).rows[0]
+        //   .price_hour;
+        providerPriceHour = providerData.price_hour;
```

Comment:
* you cant declare variables inside the switch so you need to declare ``` let providerData ``` outside the switch then give it the provider data.
```js
   [providerData] = (await getProviderDataById(providerId)).rows;

```
*  remove the comment code
* delete unused query file(getPriceHourProvider)  inside the require .